### ISSUE TITLE
API keys endpoint doesn't contain no-role keys

### ISSUE THREAD

**Issue by wolfv** [mamba-org/quetz#134]
After creating some keys, it's still empty...

**Comment by btel** [comment]
it works for me (key created with user privileges):

```
curl http://localhost:8000/api/api-keys -H "X-API-Key: ${QUETZ_API_KEY}" | jq
```

````
[
  {
    "description": "",
    "roles": [
      {
        "channel": "test-robo-3",
        "package": null,
        "role": "owner"
      },
      {
        "channel": "test-robo-4",
        "package": null,
        "role": "owner"
      },
      {
        "channel": "test-robo-5",
        "package": null,
        "role": "owner"
      },
      {
        "channel": "test-robo-6",
        "package": null,
        "role": "owner"
      },
      {
        "channel": "test-robo-7",
        "package": null,
        "role": "owner"
      },
      {
        "channel": "test-robo-8",
        "package": null,
        "role": "owner"
      }
    ],
    "key": "5RpVCJWjAQEo1GkewY_Od3SmlRg6Xrhg6CIBThbEMKI"
  }
]

**Comment by wolfv** [comment]
Interesting, I now also get a response.
I think the problem is apparent when there are _no_ channels (initial, empty state).

Btw. I currently have one API key per (mirrored) package.

**Comment by wolfv** [comment]
this is fixed with #215 correct?
